docs/getting-started/quickstart.md

```diff
@@ -173,7 +169,9 @@ node start_postgres_docker.js
    ```
   </TabItem>
     <TabItem value="linux" label="Linux">
+   Follow the [Docker Engine installation page](https://docs.docker.com/engine/install/) to install Docker on several popular Linux distributions.
```

Comment:
I really like how those steps are now combined! We can drop the :::tip I think.

Reply:
I would like to defend that tip (with my honor). I put that there last PR because I really needed it when it was my time. It would've saved me like an hour.

It was something like "How am I supposed to know there's a record in the database? Let me connect and look at the schema. Okay but how do I connect to this docker thing? How am I supposed to know that the port 5432 is forwarded to the outside?" I suppose I could've inferred it from knowing that the app talks to it but... it took a while :)